packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -28,7 +28,10 @@ export default class Sayt extends LitElement {
    * Shows a button to allow for closing SAYT manually.
    */
   @property({ type: Boolean, reflect: true }) showCloseButton = false;
-
+  /**
+   * Customizes the number of characters entered in searchbox before opening SAYT.
```

Comment:
Remember that it's also valid to read from this property, so the description should make sense in that context as well. Maybe something like this?

```suggestion
   * The minimum length of the search term required before a SAYT request will be made with it.
```